src/components/interactive-builder/modals/question/question-form/question-types/inputs/test-order/test-order-type-question.component.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, FormGroup, Stack, TextInput } from '@carbon/react';
+import { Add, TrashCan } from '@carbon/react/icons';
+import ConceptSearch from '../../../common/concept-search/concept-search.component';
+import { useFormField } from '../../../../form-field-context';
+import styles from './test-order-type-question.scss';
+
+interface SelectableOrder {
+  concept: string;
+  label: string;
+}
+
+const TestOrderTypeQuestion: React.FC = () => {
+  const { t } = useTranslation();
+  const { formField, setFormField } = useFormField();
+
+  const selectableOrders = useMemo(
+    () => formField.questionOptions?.selectableOrders || [],
+    [formField.questionOptions?.selectableOrders],
+  );
+
+  const handleConceptChange = useCallback(
+    (concept: string) => {
+      setFormField((prevField) => ({
+        ...prevField,
+        questionOptions: {
+          ...prevField.questionOptions,
+          concept,
+        },
+      }));
+    },
+    [setFormField],
+  );
+
+  const handleAddSelectableOrder = useCallback(() => {
+    const newOrder: SelectableOrder = { concept: '', label: '' };
+    setFormField((prevField) => ({
+      ...prevField,
+      questionOptions: {
+        ...prevField.questionOptions,
+        selectableOrders: [...selectableOrders, newOrder],
+      },
+    }));
+  }, [selectableOrders, setFormField]);
+
+  const handleRemoveSelectableOrder = useCallback(
+    (index: number) => {
+      const updatedOrders = selectableOrders.filter((_, i) => i !== index);
+      setFormField((prevField) => ({
+        ...prevField,
+        questionOptions: {
+          ...prevField.questionOptions,
+          selectableOrders: updatedOrders,
+        },
+      }));
+    },
+    [selectableOrders, setFormField],
+  );
+
+  const handleSelectableOrderChange = useCallback(
+    (index: number, field: keyof SelectableOrder, value: string) => {
+      const updatedOrders = selectableOrders.map((order, i) => (i === index ? { ...order, [field]: value } : order));
+      setFormField((prevField) => ({
+        ...prevField,
+        questionOptions: {
+          ...prevField.questionOptions,
+          selectableOrders: updatedOrders,
+        },
+      }));
+    },
+    [selectableOrders, setFormField],
+  );
+
+  const handleSelectableOrderConceptChange = useCallback(
+    (index: number, concept: string) => {
+      handleSelectableOrderChange(index, 'concept', concept);
+    },
+    [handleSelectableOrderChange],
+  );
+
+  return (
+    <div className={styles.testOrderContainer}>
+      <FormGroup legendText={t('concept', 'Concept')}>
+        <ConceptSearch
+          defaultConcept={formField.questionOptions?.concept}
+          onSelectConcept={(concept) => handleConceptChange(concept.uuid)}
+        />
+      </FormGroup>
```

Comment:
(suggestion) Since the concept search already has a helper text, lets not have an additional label that says `Concept`
```suggestion
        <ConceptSearch
          defaultConcept={formField.questionOptions?.concept}
          onSelectConcept={(concept) => handleConceptChange(concept.uuid)}
        />
```